rfd/00xx-kubernetes-access-cluster-discovery.md

```diff
@@ -0,0 +1,456 @@
+---
+authors: Tiago Silva (tiago.silva@goteleport.com)
+state: draft
+---
+
+# RFD XX - AWS Kubernetes Cluster Automatic discovery
+
+## Required Approvers
+
+- Engineering: `@r0mant`
+- Product: `@klizhentas || @xinding33`
+
+## What
+
+Proposes the implementation for Teleport's Kubernetes service to automatically discover and enroll EKS clusters.
+
+### Related issues
+
+- [#12048](https://github.com/gravitational/teleport/issues/12048)
+
+## Why
+
+Currently, when an operator wants to configure a new Kubernetes cluster in the Teleport, he can opt for these two methods:
```

Comment:
style nit here or below: you can use `he or she`, alternate between `he` or `she` in different sentences, or use more modern version `they` everywhere that is even more broader and simpler.